src/pages/fr/core-concepts/sharing-state.md

```diff
@@ -0,0 +1,698 @@
+:
+- **Ils sont légers.** Nano Stores expédie le strict minimum de JS dont vous aurez besoin (moins de 1 Ko) sans aucune dépendance.
+- **Ils sont indépendants du framework.** Cela signifie que le partage d'état entre les frameworks sera transparent ! Astro est basé sur la flexibilité, nous aimons donc les solutions qui offrent une expérience de développeur similaire, quelle que soit votre préférence.
+
+Pourtant, il existe un certain nombre d'alternatives que vous pouvez explorer. Ceux-ci incluent:
+- Les [Stores intégrés de Svelte](https://svelte.dev/tutorial/writable-stores)
+- [Solid signals](https://www.solidjs.com/docs/latest) en dehors d'un contexte de composant
+- [Envoi d'événements de navigateur personnalisés](https://developer.mozilla.org/fr/docs/Web/Events/Creating_and_triggering_events) entre les composants
+
+:::note[FAQ]
+
+<details>
+<summary>**🙋 Puis-je utiliser Nano Stores dans des fichiers `.astro` ou d'autres composants côté serveur ?**
+</summary>
+
+Nano Stores _peut_ être importé, écrit et lu dans les composants côté serveur, **mais nous ne le recommandons pas !** Cela est dû à quelques restrictions:
+- L'écriture dans un store à partir d'un fichier `.astro` ou [composant non hydraté](/fr/core-concepts/framework-components/#hydratation-des-composants-interactifs) n'affectera _pas_ la valeur reçue par [composants côté client] (/fr/reference/directives-reference/#client-directives).
+- Vous ne pouvez pas transmettre un Nano Store en tant que `prop` aux composants côté client.
+- Vous ne pouvez pas vous abonner aux mises à jour du store à partir d'un fichier `.astro`, puisque les composants Astro ne se rafraîchissent pas. 
+
+Si vous comprenez ces restrictions et que vous trouvez toujours un cas d'utilisation, vous pouvez essayer Nano Stores ! N'oubliez pas que les Nano Stores sont conçus pour être réactifs aux changements sur le **client** spécifiquement.
+
+</details>
+
+<details>
+<summary>**🙋 Comment les Stores de Svelte sont-ils différents de Nano Stores?**</summary>
+
+**Nano Stores et les [Stores de Svelte](https://svelte.dev/tutorial/writable-stores) sont très similaires!** En effet, [Nano Stores vous permettent d'utiliser le même raccourci `$`](https://github.com/nanostores/nanostores#svelte) pour les abonnements que vous pourriez utiliser avec Svelte stores.
+
+Si vous voulez éviter les bibliothèques tierces, [Svelte stores](https://svelte.dev/tutorial/writable-stores) sont à eux seuls un excellent outil de communication inter-îles. Néanmoins, vous préférerez peut-être Nano Stores si a) vous aimez leurs modules complémentaires pour ["objects"](https://github.com/nanostores/nanostores#maps) et [async state](https://github.com/nanostores/nanostores#lazy-stores), ou b) vous souhaitez communiquer entre Svelte et d'autres frameworks d'interface utilisateur comme Preact ou Vue.
```

Comment:
Yes I think, @MoustaphaDev is right as these aren’t the specific technical terms used by nanostores (they say “maps” and “lazy stores” instead).

```suggestion
Si vous voulez éviter les bibliothèques tierces, [Svelte stores](https://svelte.dev/tutorial/writable-stores) sont à eux seuls un excellent outil de communication inter-îles. Néanmoins, vous préférerez peut-être Nano Stores si a) vous aimez leurs modules complémentaires pour les [objets](https://github.com/nanostores/nanostores#maps) et [l'état asynchrone](https://github.com/nanostores/nanostores#lazy-stores), ou b) vous souhaitez communiquer entre Svelte et d'autres frameworks d'interface utilisateur comme Preact ou Vue.
```

Reply:
> Yes I think, @MoustaphaDev is right as these aren’t the specific technical terms used by nanostores (they say “maps” and “lazy stores” instead).

Nice revisions!